Snowflake.Data/Core/ArrowResultChunk.cs

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+using System;
+using System.Text;
+using Apache.Arrow;
+using Apache.Arrow.Types;
+
+namespace Snowflake.Data.Core
+{
+    internal class ArrowResultChunk : IResultChunk
+    {
+        public RecordBatch RecordBatch { get; set; }
+
+        private int _rowCount;
+        private int _colCount;
+        private int _chunkIndex;
+
+        public ArrowResultChunk(RecordBatch recordBatch)
+        {
+            RecordBatch = recordBatch;
+            
+            _rowCount = recordBatch.Length;
+            _colCount = recordBatch.ColumnCount;
+            _chunkIndex = 0;
+        }
+
+        public UTF8Buffer ExtractCell(int rowIndex, int columnIndex)
+        {
+            var column = RecordBatch.Column(columnIndex);
+
+            string s;
+            switch (column.Data.DataType.TypeId)
+            {
+                case ArrowTypeId.Int32: 
+                    s = ((Int32Array)column).GetValue(rowIndex).ToString();
```

Comment:
what is done here is converting int value to string and later converting string to array of bytes.
Maybe more efficient would be just BitConverter.GetBytes(...)?

Reply:
it is temporary solution, full implementation will be done in SNOW-893834 (maybe with BitConverter(